structures/message.js

```diff
@@ -0,0 +1,24 @@
+const s = require('superstruct');
+const { ZuluDateTimeStruct } = require('./lib');
+const uuid = require('uuid');
+const isUuid = require('is-uuid');
+
+const Uuid = s.define('Uuid', isUuid.v4);
+
+const Message = s.type({
+    id: s.defaulted(Uuid, uuid.v4()),
```

Comment:
```suggestion
    id: s.defaulted(Uuid, uuid.v4),
```
to avoid generating the same uuid every time